circle.yml

```diff
@@ -17,13 +30,38 @@ jobs:
       - run:
           name: "NDK Fix: https://github.com/android-ndk/ndk/issues/188"
           command: sudo apt-get install file
+      - run:
+          name: Setup emulator
+          command: |
+            sdkmanager "system-images;android-19;default;armeabi-v7a"
+            echo "no" | avdmanager create avd -n test -k "system-images;android-19;default;armeabi-v7a"
+      - run:
+          name: Launch emulator
+          command: emulator -avd test -no-audio -no-window
+          background: true
+      - run:
+          name: Wait for emulator
+          command: |
+            wget --quiet --output-document=android-wait-for-emulator https://raw.githubusercontent.com/travis-ci/travis-cookbooks/master/community-cookbooks/android-sdk/files/default/android-wait-for-emulator
```

Comment:
I'm a bit wary of downloading something and running it. Could we instead add a snapshot of this script to our repo?

Reply:
Yes, I think in the long run we should we use fresco own docker instead of the official one. In this way we can also remove download ndk part.

Reply:
Sure, I might work on that at some point. In the meantime we should still add this script to our repo and call it locally. Do you think you can do that?